beeline-http-client/src/Beeline/HTTP/Client.hs

```diff
@@ -1,5 +1,11 @@
 {-# OPTIONS_GHC -Wno-missing-import-lists #-}
 
+{- |
+Copyright : Flipstone Technology Partners 2023-2025
+License   : MIT
+
+@since 0.1.3.0
```

Comment:
Why add the minor/patch version numbers to `since` annotations? It could give readers the impression that the repo doesn't follow PVP. Since the library hasn't been published to Hackage yet, isn't it premature to version at all? As far as I understand, the version number is currently only incremented to make sure the build cache is invalidated. It would make sense to me to set all `since` annotations to the initial Hackage published version (which doesn't exist yet), since I can't really imagine them being used before then.

Reply:
Well modules/functions can be added in a minor version per the pvp:
>  Otherwise, if only new bindings, types, classes, non-orphan instances or modules (but see below) were added to the interface, then A.B MAY remain the same but the new C MUST be greater than the old C. 

So they absolutely do belong in since annotations.

Further, we should just be in the practice of versioning. So adding them now, is go be building up that muscle. When it comes time to publish to hackage we can adjust the since annotations to the first published version. That's what we did with orville.

Reply:
Oh, I somehow keep forgetting this part of PVP. Thanks for the explanation.

Reply:
> Oh, I somehow keep forgetting this part of PVP. Thanks for the explanation.

No problem at all! The PVP rules can be hard to remember sometimes. For me at least, the two numbers for major version, feels atypical and thus more confusing/harder to remember. 

Also, I can see an argument both ways for the patch level number (the `D` in the `A.B.C.D`) being included in the since annotations. My reading of the PVP is that it _must_ always be whatever the lowest number is for a given `A.B.C` (in practice that would be 0 but I don't think it _must_ be 0, we _could_ start them with 1). So then it _could_ be left out _if_ you remember the PVP rules. But I think it's clearer to just include the full version number and not require any additional information.